cmd/livepeer/livepeer.go

```diff
@@ -397,21 +410,10 @@ func main() {
 		drivers.NodeStorage = drivers.NewMemoryDriver(n.GetServiceURI())
 	}
 
-	if n.NodeType == core.BroadcasterNode {
-		if len(orchAddresses) > 0 {
-			n.OrchestratorPool = discovery.NewOrchestratorPool(n, orchAddresses)
-		} else if !*offchain {
-			n.OrchestratorPool = discovery.NewDBOrchestratorPoolCache(n)
-		}
-		if n.OrchestratorPool == nil {
-			glog.Errorf("No orchestrator specified; transcoding will not happen")
-		}
-	}
-
 	//Create Livepeer Node
 
 	// Set up logging
-	if !*noIPFSLogFiles {
+	if !*logIPFS {
```

Comment:
Perhaps it would be clearer to default `logIPFS` to true and check for `if *logIPFS {}` here?

Reply:
Ah this actually should have been `if *logIPFS {`. Fixed.

Should we set the default to `true` even with IPFS being unused?

Reply:
Hm right - I think it makes sense to default to `false` right now since IPFS is unused, but if/when we start using IFPS again then it might make sense to default it to `true` to avoid IPFS log messages adding noise to the Livepeer node log output.